ETC_en.tsv

```diff
@@ -5067,10 +5067,10 @@ ETC_20150317_005066	Oak Pattern
 ETC_20150317_005067	Rough Pottery
 ETC_20150317_005068	Navy
 ETC_20150317_005069	Tu-Tutu...! Help me!!!
-ETC_20150317_005070	Please defeat Unknocker!
+ETC_20150317_005070	Please defeat the Unknocker!
```

Comment:
"Please defeat Unknocker!" or "Defeat Unknocker!" (depending if this piece of line is shown in NPC dialogue or quest UI) 

Unknocker is a boss monster.